test/fixtures/echo-close-check.js

```diff
@@ -26,9 +26,15 @@ const fs = require('fs');
 
 process.stdout.write('hello world\r\n');
 
-var stdin = process.openStdin();
+const stdin = process.openStdin();
+
+let current;
+common.hijackStdout(common.mustCall(function(data) {
```

Comment:
I don't this this is an interesting use of `hijackStdout`, it just tests the mechanics of `hijackStdout`.
BTW: you __should__ do that in explicitly in `test/parallel/test-common.js`